db/migrate/20230320144827_create_records_table.rb

```diff
@@ -0,0 +1,15 @@
+class CreateRecordsTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :records do |t|
+      t.string :name, null: false
+      t.decimal :amount, null: false, default: 0.00
+      t.integer :user_id, null: false
+      t.integer :category_id, null: false
+      t.timestamps
+    end
+    add_foreign_key :records, :users, column: :user_id
```

Comment:
- [ ] one of the requirements is not to change the name of the column in the database so you should name the foreign key as `authored_id ` as shown in the diagram so please try to do so 
- Screenshot of the requirements 
<img width="931" alt="Screenshot 2023-03-23 at 12 31 26" src="https://user-images.githubusercontent.com/61790587/227205005-8dfa9503-1f48-4ab8-b357-6e776ec4d94f.png">